doc/sphinx/source/esmvaldiag/observations.rst

```diff
@@ -0,0 +1,283 @@
+  already and which takes care of cmorizing the coordinates of the current
+  variable if necessary (e.g., longitudes ranging from -180 to 180 degrees
+  instead of 0 to 360 degrees). 
+- ``fout``: the filepath and filename of the output file are set here. The path
+  is taken from the configuration file (see Section 2) that is necessary to run
+  the cmorizing script, and the filename is put together from the
+  information given in the first part of the script, following the rules for
+  filenames so that the ESMValTool can read in the files. 
+
+The script as it is detailed here would only be able to correct some minor
+problems with the coordinates (e.g. latitudes in the wrong order, longitudes in
+the wrong order, etc.). Everything else will have to be added to the script for
+it to deal with it. There are many cmorizing scripts available in
+`/esmvaltool/utils/cmorizers/obs/
+<https://github.com/ESMValGroup/ESMValTool/blob/version2_development/esmvaltool/utils/cmorizers/obs/>`_
+where solutions to many kinds of format issues with observational data are
+addressed. Most of these scripts are written in NCL, but there more and more
+examples for Python-based cmorizing scripts become available.
+
+How much cmorizing an observational data set needs is strongly dependent on
+the original NetCDF file and how close the original formatting already is to
+the strict CMOR standard. 
+
+5. Run the cmorizing script
+===========================
+
+In order to actually run the cmrorizing script, you have to use the following statement:
+
+.. code-block:: console
+
+ cmorize_obs -c <config-user.yml> -o <dataset-name>
+
+This call only works, of course, if you are already in the folder in which also
+the ``config-user.yml`` is stored.
+
+.. note::
+
+   The output path given in the configuration file is the path where
+   your cmorized dataset will be stored. The ESMValTool will create a folder
+   with the correct tier information (see Section 2) if that tier folder is not
+   already available, and then a folder named after the data set. In this
+   folder the cmorized data set will be stored as a netCDF file. 
+
+Your run was successful if a netCDF file was produced in your output directory,
+and if at some point the output on your screen shows an info line similar to
+this and no additional error message is shown (note: this is the example for
+the example data set ``xch4``): 
+
+.. code-block:: console
+
+  INFO    Processing xch4 (Amon)
+
+6. Naming convention of the observational data files
+====================================================
+
+For the ESMValTool to be able to read the observations from the NetCDF file,
+the file name needs a very specific structure and order of information parts
+(very similar to the naming convention for observations in ESMValTool
+v1.0). The file name will be automatically correctly created if a cmorizing
+script has been used to create the netCDF file. If the file with the
```

Comment:
I would delete the rest of the paragraph, from `If the file with...`.

Reply:
I removed the rest of the paragraph.